kofta/package.json

```diff
@@ -11,6 +11,7 @@
     "@types/jest": "^26.0.15",
     "@types/node": "^12.0.0",
     "@types/react-dom": "^16.9.8",
+    "dateformat": "^4.5.1",
```

Comment:
can you use date-fns so we use the same date lib for everything

Reply:
Okay sure.